codegen.yml

```diff
@@ -1,6 +1,6 @@
 
 generates:
-  ./src/generated/graphql-shop-api-types.ts:
+  ./src/ui/generated/graphql-shop-api-types.ts:
```

Comment:
Place generated inside `ui` dir, because Admin UI Angular compilation can't access anything outside the `ui` directory